ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
After running this code,

dash@Dash:~/development_ws$ rosparam list | grep prefix
/rviz_Dash_5364_7703676098926058836/arm_prefix

which obviously, by the code shows it's in RViz namespace,

since the URKinematics looks for the arm_prefix parameter, you have to wonder where else could it be getting set from? 

When I remove that LOC from moveit_rviz.launch:

[ERROR] [1424975775.985587915, 21.224000000]: Kin chain provided in model doesn't contain standard UR joint 'shoulder_lift_joint'.
[ERROR] [1424975775.985692288, 21.224000000]: Kinematics solver of type 'ur_kinematics/UR5KinematicsPlugin' could not be initialized for group 'ur5_arm'
[ERROR] [1424975775.986159403, 21.224000000]: Kinematics solver could not be instantiated for joint group ur5_arm.

Even when adding it to another place in moveit_rviz.launch, or adding it to kinematics.yaml, I still received that error.

The only way to fix it is to add that parameter at that specific location. I'm not sure exactly how moveit_plugins are loaded or where they're loaded into.. I've taken a look but haven't seen anything showing where moveit plugins are loaded in yet.